src/main/java/frc/robot/RobotContainer.java

```diff
@@ -143,10 +161,11 @@ Sysid button commands, commented out (I like keeping this commented because
                                 Constants.Landmarks.reefFacingAngleRed[5].getRadians())))));
     Trigger rightBumper = joystick.rightBumper();
 
-    rightBumper.onTrue(new Dealgaenate(ArmSubsystem.getInstance()));
-    rightBumper.onFalse(
-        new ArmToAngleCmd(Constants.Arm.RETRACTED_ANGLE, ArmSubsystem.getInstance()));
-    joystick.y().whileTrue(JamesHardenMovement.toClosestRightBranch(driveTrain, redside));
+    // rightBumper.onTrue(new Dealgaenate(ArmSubsystem.getInstance()));
+
+    // rightBumper.onFalse(
+    //     new ArmToAngleCmd(Constants.Arm.RETRACTED_ANGLE, ArmSubsystem.getInstance()));
+    // joystick.y().whileTrue(JamesHardenMovement.toClosestRightBranch(driveTrain, redside));
```

Comment:
Leave a comment explaining why we need this code, or remove it if we don't need it.